js/server/tests/shell/shell-collection-properties-cluster-spec.js

```diff
@@ -236,4 +242,19 @@ describe('Replication factor constraints', function() {
     it('should allow to create a collection with more replicas than dbservers when explicitly requested', function() {
         db._create(cn1, {replicationFactor: 5}, {enforceReplicationFactor: false});
     });
+
+    it('check replication factor of system collections', function() {
+        ["_appbundles", "_apps", "_aqlfunctions", "_frontend", "_graphs",
+         "_iresearch_analyzers", "_jobs", "_modules", "_queues", "_routing",
+         "_statistics" , "_statistics15" , "_statisticsRaw" ,"_users"
+        ].forEach(name => {
+          expect(db[name].properties()['replicationFactor']).to.equal(2);
```

Comment:
This test is dangerous.  `_statistics*` will be created lazily. I think the same holds true for `_apps` or `_appbundles`.
Can you encapsulate those in an `if(db[name]) { expect [....] }`
You can also ensure here that all follow `_graphs` except `_graphs` ;)